_posts/2024-10-09-call-for-editors.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+date: 2024-10-09 00:00
+title:  "Call for editors"
+category:
+- announcements
+---
+
+Once again, we're looking to grow our editorial team at JOSS!
+
+Since our launch in [May 2016](https://www.arfon.org/announcing-the-journal-of-open-source-software), our [existing editorial team](http://joss.theoj.org/about#editorial_board) has handled nearly 4500 submissions (~2700 published at the time of writing, 340 under review) and the demand from the community continues to be strong. JOSS now consistently publishes a little over one paper per day, and we see no sign of this demand dropping.
```

Comment:
I don't think the "a little over" is right. We're at about 1.3 - 1.5 papers per day, so I might say "40-45 papers per month, and this is gradually increasing"